tools/build.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python
+# Copyright 2018 the Deno authors. All rights reserved. MIT license.
+import argparse
+import os
+import sys
+from os.path import join
+from util import run
+import distutils.spawn
+
+parser = argparse.ArgumentParser(description='')
```

Comment:
It'd be nice if it would reject unknown arguments.
I `python tools\build.py` and it happily started building (but that's not so great because it also reconfigured out/debug with the default args...)

Reply:
Hmm - what were you thinking? This was the intended behavior. (From this point forward if we do custom builds, we should call them something other than debug or release.)

Reply:
Sorry - my bad.
I did 'python tools/build.py --helo'
The unrecognized argument should not have been ignored.

Reply:
Oh... but I still am not sure how to deal with this. Currently extra args are passed thru to nina - which is useful for doing things like `tools/build.py -j1`... Can we deal with this later?